packages/svelte-ux/src/lib/components/MenuButton.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
-  import type { ComponentProps } from '$lib/types/index.js';
+  import type { ComponentProps } from '../types/typeHelpers.js';
```

Comment:
Good idea! Took care of that in [fc7f883](https://github.com/techniq/svelte-ux/pull/300/commits/fc7f883907d3c049f0983542cc4823a2a3448e61)

To avoid breaking changes, I updated the existing def/export to alias the native ComponentProps type helper and added some deprecation notes that look real nice on hover 😁

![](https://github.com/techniq/svelte-ux/assets/5913254/3d36d879-34fe-4b2d-9424-02a69c30f1d1)

![](https://github.com/techniq/svelte-ux/assets/5913254/346f5767-32a4-4bb6-bb5e-0b54d8ee356e)